<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="QTLSYS2-34" name="pack.APR_Calculationcrome" tests="5" failures="0" timestamp="26 May 2020 14:52:53 GMT" time="38526.395" errors="2">
  <testcase name="executSessionFour" time="15.391" classname="pack.APR_Calculationcrome">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pack.Excel.getLastRow(Excel.java:83)
at pack.APR_Calculationcrome.executSessionFour(APR_Calculationcrome.java:1032)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- executSessionFour -->
  <testcase name="executSessionFIve" time="4749.166" classname="pack.APR_Calculationcrome"/>
  <testcase name="executSessionThree" time="13670.494" classname="pack.APR_Calculationcrome"/>
  <testcase name="executSessionOne" time="15.417" classname="pack.APR_Calculationcrome">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pack.Excel.getLastRow(Excel.java:83)
at pack.APR_Calculationcrome.executSessionOne(APR_Calculationcrome.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- executSessionOne -->
  <testcase name="executeSessionTwo" time="20075.928" classname="pack.APR_Calculationcrome"/>
</testsuite> <!-- pack.APR_Calculationcrome -->
